src/app/portlets/dot-edit-page/content/services/dot-edit-content-html/dot-edit-content-html.service.spec.ts

```diff
@@ -147,7 +147,28 @@ describe('DotEditContentHtmlService', () => {
         });
     });
 
-    it('should relocate contentlet', () => {
+    it('should render edit internal contentlet', () => {
+        spyOn(this.dotEditContentHtmlService, 'renderEditedContentlet');
+
+        this.dotEditContentHtmlService.currentContentlet = {
+            identifier: '444',
+            inode: '555'
+        };
+
+        this.dotEditContentHtmlService.contentletEvents$.next({
+            name: 'save',
+            data: {
+                identifier: '456'
+            }
+        });
+
+        expect(this.dotEditContentHtmlService.renderEditedContentlet).toHaveBeenCalledWith({
+            identifier: '444',
+            inode: '555'
+        });
+    });
+
+    it('should render relocated contentlet', () => {
```

Comment:
don't we need a test to check this?

https://github.com/dotCMS/core-web/pull/667/files#diff-e2760b805d923abc4137bf2e3ddf0ee9R66